tutorials/tutorials.batch

```diff
@@ -13,98 +13,74 @@
 ~/bin/highlight-pointer -r 10 --key-quit q &
 mkdir /tmp/msui_tutorials
 export MSUI_CONFIG_PATH=/tmp/msui_tutorials
+export XDG_CONFIG_PATH=/tmp/msui_tutorials
 
 ##########################################################################
 # wms tutorial
-$HOME/mambaforge/envs/mssdev/bin/python tutorial_wms.py
+$HOME/Miniforge/envs/mssdev/bin/python tutorial_wms.py
```

Comment:
This should just rely on the right python being in the PATH, instead of making so many assumptions, IMO.

Reply:
I know, step by step. This gets later worked on.